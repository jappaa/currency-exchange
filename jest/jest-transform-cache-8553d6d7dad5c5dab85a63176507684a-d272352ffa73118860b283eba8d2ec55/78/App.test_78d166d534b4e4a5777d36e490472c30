'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _enzyme = require('enzyme');
var _reduxMockStore = require('redux-mock-store');var _reduxMockStore2 = _interopRequireDefault(_reduxMockStore);
var _App = require('../App');var _App2 = _interopRequireDefault(_App);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

describe('App', function () {
  var createStore = (0, _reduxMockStore2.default)();
  var component = void 0;

  beforeEach(function () {
    component = (0, _enzyme.shallow)(_react2.default.createElement(_App2.default, null));
    var store = createStore(state);
  });

  it('mathces its snapshot', function () {
    expect(component).toMatchSnapshot();
  });
});