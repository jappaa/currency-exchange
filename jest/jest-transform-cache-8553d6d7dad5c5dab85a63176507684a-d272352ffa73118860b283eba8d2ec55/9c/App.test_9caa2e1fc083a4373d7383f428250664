'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _enzyme = require('enzyme');
var _App = require('../App');var _App2 = _interopRequireDefault(_App);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

describe('App', function () {
  var component = void 0;

  beforeEach(function () {
    component = (0, _enzyme.shallow)(_react2.default.createElement(_App2.default, null));
  });

  it('mathces its snapshot', function () {
    expect(component).toMatchSnapshot();
  });

  it('uses correct containers', function () {
    expect(component.find('Connect(Main)').exists()).toBe(true);
  });
});