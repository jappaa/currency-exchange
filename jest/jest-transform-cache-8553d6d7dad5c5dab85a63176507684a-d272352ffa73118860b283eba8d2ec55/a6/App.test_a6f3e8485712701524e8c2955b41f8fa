'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _reduxMockStore = require('redux-mock-store');var _reduxMockStore2 = _interopRequireDefault(_reduxMockStore);
var _reactRedux = require('react-redux');
var _enzyme = require('enzyme');
var _App = require('../App');var _App2 = _interopRequireDefault(_App);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

describe('App', function () {
  var createStore = (0, _reduxMockStore2.default)();
  var component = void 0;

  beforeEach(function () {
    var store = createStore();
    component = (0, _enzyme.mount)(_react2.default.createElement(_reactRedux.Provider, { store: store }, ' ', _react2.default.createElement(_App2.default, null), ' ')).find('App');
  });

  it('mathces its snapshot', function () {
    expect(component).toMatchSnapshot();
  });
});