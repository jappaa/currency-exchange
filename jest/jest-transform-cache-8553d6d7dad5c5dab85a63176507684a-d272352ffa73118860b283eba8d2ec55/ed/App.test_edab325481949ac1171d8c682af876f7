'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _enzyme = require('enzyme');
var _reduxMockStore = require('redux-mock-store');var _reduxMockStore2 = _interopRequireDefault(_reduxMockStore);
var _App = require('../App');var _App2 = _interopRequireDefault(_App);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

describe('App', function () {
  var createStore = (0, _reduxMockStore2.default)();
  var component = void 0;

  beforeEach(function () {
    var store = createStore();
    component = mount(
    _react2.default.createElement(Provider, { store: store }, ' ', _react2.default.createElement(TodoList, null), ' different components ')).
    find('App');
  });

  it('mathces its snapshot', function () {
    expect(component).toMatchSnapshot();
  });
});