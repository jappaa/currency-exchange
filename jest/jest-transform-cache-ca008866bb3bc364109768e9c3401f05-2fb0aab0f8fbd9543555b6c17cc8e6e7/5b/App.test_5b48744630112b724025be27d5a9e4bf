'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _reduxMockStore = require('redux-mock-store');var _reduxMockStore2 = _interopRequireDefault(_reduxMockStore);
var _reactRedux = require('react-redux');
var _enzyme = require('enzyme');
var _App = require('../__mocks__/App');var _App2 = _interopRequireDefault(_App);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

describe('App', function () {
  var component = void 0;
  beforeEach(function () {
    component = (0, _enzyme.shallow)(_react2.default.createElement(_App2.default, null));
  });

  it('mathces its snapshot', function () {
    expect(component).toMatchSnapshot();
  });
});