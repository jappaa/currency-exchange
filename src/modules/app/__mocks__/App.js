import React from 'react'

// smoke test
const App = () => <h1>App</h1>
export default App
