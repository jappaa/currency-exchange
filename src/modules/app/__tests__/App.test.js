import React from 'react'
import configureMockStore from 'redux-mock-store'
import { Provider } from 'react-redux'
import { shallow, mount } from 'enzyme'
import App from '../__mocks__/App'

describe( 'App', () => {
  let component
  beforeEach( () => {
    component = shallow( <App /> )
  } )

  it( 'mathces its snapshot', () => {
    expect( component ).toMatchSnapshot()
  } )
} )
