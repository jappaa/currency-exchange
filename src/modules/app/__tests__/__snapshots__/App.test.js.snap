// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App mathces its snapshot 1`] = `
<h1>
  App
</h1>
`;
